router/auth.py

```diff
@@ -0,0 +1,45 @@
+from fastapi import APIRouter, Depends, HTTPException, Response
+from fastapi.security import OAuth2PasswordRequestForm
+from auth.auth_handler import check_password, check_user, create_access_token, get_current_user
+from auth.cookie import OAuth2PasswordBearerWithCookie
+import sys
+from decouple import config
+from pathlib import Path
+
+file = Path(__file__).resolve()
+package_root_directory = file.parents[1]
+sys.path.append(str(package_root_directory))
+from models import RefereeIdBody, SportScheduleBody
+from utils import error_handler
+
+router = APIRouter(prefix='/api/auth',
+                   tags=["auth"],
+                   responses={ 404: {"description": "Not found"}})
+
+oauth2_scheme = OAuth2PasswordBearerWithCookie(tokenUrl="api/auth/token")
+
+COOKIE_SECURE = config("SECURE", default=False, cast=bool)
+
+@error_handler
+@router.post('/token', status_code=201)
+async def login(response: Response, form_data:OAuth2PasswordRequestForm = Depends()):
```

Comment:
why it **asyncronous** now? we didnt need `async` and `await` now because we use pymongo